vim/plugin-managers.md

```diff
@@ -0,0 +1,26 @@
+# Plugin Managers
+
+Today I learned that vim has two main plugin managers:
+
+1. Tim Pope<sup>1</sup>'s [Pathogen][1]
+1. gmarik's [Vundle][2]
```

Comment:
Pretty sure this should be "**2**. gmarik's [Vundle][2]". Interestingly, the raw file renders correctly...